src/mdx-plugins/rehype-snippets.js

```diff
@@ -60,6 +60,8 @@ tcT> int lwb(V<T>& a, const T& b) { return int(lb(all(a),b)-bg(a)); }
 
 // loops
 #define FOR(i,a,b) for (int i = (a); i < (b); ++i)
+#define FORE(i,a,b) for (int i = (a); i <= (b); ++i)
```

Comment:
I'd prefer it if you didn't change this. Instead, you can include the template you're using in `Binary_Jump.mdx`.